content/docs/guides/ha_scale/perf-scale.md

```diff
@@ -108,8 +117,39 @@ There is not much difference between pod-rollout time and stable time in terms o
 
 By looking at the collected metrics, in our testing scenarios, resource consumption of data plane sidecars does not change much along with the mesh size. At stable time, the average CPU usage is close to 0 and average memory usage is around 50MB. CPU usage is related to request processing and memory usage is related the size of Envoy configuration stored locally. These two factors do not change with the mesh scale.
 
-It is also worth mentioning that enabling permissive mode will cause data plane to use much more memory, as each sidecar needs to keep the connection configuration between all other services. The chart below shows the sidecar memory usage when 200 and 400 pods are added in a permissive-mode enabled mesh. Compared to the non-permissive mode, the memory usage is more than doubled. Similar things happen on the control plane.
+### Other Scenarios
+
+#### Permissive mode
+
+When permissive mode is enabled in OSM, services are allowed to communicate with any other services in the mesh without additional traffic policies. This is implemented by adding an allow-all wildcard RBAC policy in the inbound listener and all known services as outbound clusters in the Envoy sidecar. The same load test described above is performed on a service mesh with permissive mode enabled.
+
+On the control plane, the CPU and memory usage both increase compared to non-permissive mode.
```

Comment:
which are you saying:
- CPU and memory usage is higher when running permissive mode?
- CPU and memory usage rates increased at the same rate between permissive and non-permissive mode so their overall usage is the same?